projects/ion/src/lib/picker/date-picker/date-picker-calendar/date-picker-calendar.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="container-date-picker-calendar">
```

Comment:
Try to use semantic HTML.

```suggestion
<section class="container-date-picker-calendar">
```